exec/java-exec/src/main/java/org/apache/drill/exec/store/pcap/PcapRecordReader.java

```diff
@@ -100,14 +104,14 @@ public void setup(final OperatorContext context, final OutputMutator output) thr
 
       this.output = output;
       this.buffer = new byte[100000];
-      this.in = new FileInputStream(inputPath);
+      this.in = fs.open(pathToFile);
       this.decoder = new PacketDecoder(in);
       this.validBytes = in.read(buffer);
       this.projectedCols = getProjectedColsIfItNull();
       setColumns(projectedColumns);
     } catch (IOException io) {
       throw UserException.dataReadError(io)
-          .addContext("File name:", inputPath)
+          .addContext("File name:", pathToFile.toString())
```

Comment:
It's better you use `pathToFile.toUri().getPath()`.

Reply:
Thanks, I will replace it.